queries.sql

```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * from animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Pikachu', 'Agumon');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals where neutered = true;
+SELECT name FROM animals WHERE name NOT IN ('Gabumon');
+SELECT * FROM animals WHERE weight_kg >= 10.6 AND weight_kg <= 17.3;
```

Comment:
As you need to, you should add screenshots of all your queries in the pull request description of your project. This is to help ensure that you run the queries with success. 😄